### PR TITLE
Add AWS CodeCommit examples for push request and also README to configure AWS CodeCommit

### PR DESCRIPTION
**Pull request by savitaashture**
# Changes

Add README and samples to integrate triggers with AWS codecommit

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) if any changes are user facing, including updates to minimum requirements e.g. Kubernetes version bumps
- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [ ] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including functionality, content, code)
- [ ] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [ ] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings). See some examples of [good release notes](https://github.com/tektoncd/community/blob/main/standards.md#good-release-notes).
- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Review by khrm — COMMENTED** [review]
/lgtm

**Comment by savitaashture** [comment]
@dibyom PR is ready to review PTAL
Thank you

File: examples/v1beta1/awscodecommit/push/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+   ![img.png](images/createsubscriptionsuccess.png)
+   3. Once **Create subscription** is success click on **Request confirmation**
+   ![img.png](images/requestconfirmation.png)
+      a. Then go to eventlistener pod
+
+      i. oc logs -f <pod-name>
+
+      ii. Search for SubscribeURL and copy value
+
+            "SubscribeURL\" : \"https://sns.us-east-2.amazonaws.com/?Action=ConfirmSubscription&TopicArn=arn:aws:sns:us-east-2:361754793035:demo-acc-tekton&Token=2336412f37fb687f5d51e6e2425c464de257e9a5924e2fd4cf343ac0f1dcdb40cc7dad021f86dcf2af98fbabf59b6314b975e5d5bfc673924522ab99cb9b45b824a2795d81328e045db0148571c0ba451648f60157690c2c377eb41f58a7d663c15a36654beda27c295f9f90cc8a0f46096aea96a40f9c4ca0b1ec1092e32116\"
```
Comment (dibyom) [inline examples/v1beta1/awscodecommit/push/README.md:55]: non-blocking: is this an active subscription? we may want to change it to something fake

**Review by dibyom — APPROVED** [review]

File: examples/v1beta1/awscodecommit/push/README.md (hunk 1 above)
Comment (savitaashture) [reply examples/v1beta1/awscodecommit/push/README.md:55]: It no more works

So is that okay if i keep it as it is ?

**Pull request merged**